src/linearmixedmodel.jl

```diff
@@ -414,6 +414,14 @@ function createAL(reterms::Vector{<:AbstractReMat{T}}, Xy::FeMat{T}) where {T}
     return identity.(A), identity.(L)
 end
 
+function StatsAPI.cooksdistance(model::LinearMixedModel)
+    r = residuals(model)
+    _, p, _, _ = size(model)
+    mse = dispersion(model, true)
+    hii = leverage(model)
+    return @. (r / (1 - hii))^2 * hii / (mse * p)
```

Comment:
An interesting future addition could be conditional Cook's distance as described in this paper: https://www.vvsor.nl/wp-content/uploads/2020/06/KM2000063003.pdf